<launch>
  <param name="ec_cluster_tolerance" value="0.03" />
  <param name="ec_min_cluster_size" value="700" />
  <param name="ec_max_cluster_size" value="7500" />
  <param name="distance_above_plane" value="0.01" />
  <param name="crop_min_x" value="-1.0" />
  <param name="crop_max_z" value="1.5" />
  <param name="crop_max_x" value="6.5" />
  <param name="crop_min_z" value="0.7" />
  <param name="crop_max_y" value="1.05" />
  <param name="crop_min_y" value="-1.05" />
  <node pkg="perception" type="table_recognizer_main" name="table_recognizer_main" output="screen">
    <remap from="cloud_in" to="/head_camera/depth_registered/points" />
  </node>
</launch>